OpenShift template failed to apply

```
11:51:39 [2021-01-11 17:51:39] [ERROR] [saasherder.py:_process_template:328] - [https://github.com/osbuild/image-builder//templates/image-builder.yml:main] error fetching template: while parsing a block collection
11:51:39   in "<byte string>", line 170, column 11:
11:51:39               - image: "${COMPOSER_IMAGE}:${CO ... 
11:51:39               ^
11:51:39 expected <block end>, but found '?'
11:51:39   in "<byte string>", line 185, column 11:
11:51:39               volumeMounts:
11:51:39               ^
```

--- a/templates/image-builder.yml
+++ b/templates/image-builder.yml
@@ -167,21 +167,21 @@ objects:
           name: osbuild-composer
       spec:
         containers:
-          - image: "${COMPOSER_IMAGE}:${COMPOSER_TAG}"
-            name: osbuild-composer
-            ports:
-            - name: api
-              containerPort: 443
-              protocol: TCP
-            env:
-              - name: OSBUILD_URL
-                value: "${OSBUILD_URL}"
-              - name: OSBUILD_CERT_PATH
-                value: "${OSBUILD_CERT_PATH}"
-              - name: OSBUILD_KEY_PATH
-                value: "${OSBUILD_KEY_PATH}"
-              - name: OSBUILD_CA_PATH
-                value: "${OSBUILD_CA_PATH}"
+        - image: "${COMPOSER_IMAGE}:${COMPOSER_TAG}"
+          name: osbuild-composer
+          ports:
+          - name: api
+            containerPort: 443
+            protocol: TCP
+          env:
+            - name: OSBUILD_URL
+              value: "${OSBUILD_URL}"
+            - name: OSBUILD_CERT_PATH
+              value: "${OSBUILD_CERT_PATH}"
+            - name: OSBUILD_KEY_PATH
+              value: "${OSBUILD_KEY_PATH}"
+            - name: OSBUILD_CA_PATH
+              value: "${OSBUILD_CA_PATH}"
           volumeMounts:
             - name: composer-secrets
               mountPath: "/etc/osbuild-composer"

